tools/env

```diff
@@ -4,7 +4,16 @@ set -eu
 export CLUSTER_NAME=${CLUSTER_NAME:-testbed}
 export LOADBALANCER_IP_NAME=${LOADBALANCER_IP_NAME:-testbedip}
 export KUBERNETES_PLATFORM=${KUBERNETES_PLATFORM:-gke}
-export TB_GITHUB_ORG_NAME=$(git config --get remote.origin.url | cut -d ":" -f 2 | cut -d "/" -f 1)
+
+GET_REMOTE_ORIGIN_URL=$(git config --get remote.origin.url)
```

Comment:
This variable name may be preferably `REMOTE_ORIGIN_URL`.

Reply:
I fixed it.